infrastructure/application/cloudflare/snippets/validate_jwt.js

```diff
@@ -0,0 +1,41 @@
+export default {
+  async fetch(request) {
+    const API_URL_EXT = "https://api.editor.planx.dev";
+
+    const getJWT = (request) => {
+      const authHeader = request.headers.get("authorization");
+      const authHeaderJWT = authHeader?.match(/^Bearer (\S+)$/)?.[1];
+      if (authHeaderJWT) return authHeaderJWT;
+
+      const cookies = request.headers.get("cookie");
+      const cookieJWT = cookies?.match(/(?:^|;\s*)jwt=([^;]*)(?:;|$)/)?.[1];
```

Comment:
```suggestion
      const cookieJWT = cookies?.match(/(?:^|;\s*)jwt=([^;]+)(?:;|$)/)?.[1];
```

maybe use `+` here instead since we need a non-zero length string?